vision/datasets/imageloader_test.go

```diff
@@ -47,3 +86,71 @@ func TestImageTgzLoader(t *testing.T) {
 	_, e = BuildLabelVocabularyFromTgz("no file")
 	a.Error(e)
 }
+
+func TestImageTgzLoaderHeavy(t *testing.T) {
+	if os.Getenv("GOTORCH_TEST_IMAGE_TGZ_PATH") == "" {
```

Comment:
Add a comment here on how can we generate such a big tarball?  For example, download it from somewhere and/or shuffle it using `tools/tarball_{split|merge}`?

Reply:
done